packages/paste-website/src/styles/docSearch.ts

```diff
@@ -1,161 +1,112 @@
 import {css} from '@twilio-paste/styling-library';
+import type {CSSObject} from '@twilio-paste/styling-library';
+import {pasteBaseStyles} from '@twilio-paste/theme';
+import type {GenericThemeShape} from '@twilio-paste/theme';
 
+export const docSearchVariable = (theme: GenericThemeShape): CSSObject => {
+  return {
+    ':root': {
+      '--docsearch-primary-color': theme.backgroundColors.colorBackgroundPrimaryStronger,
+      '--docsearch-text-color': theme.textColors.colorText,
+      '--docsearch-spacing': theme.space.space60,
+      '--docsearch-icon-stroke-width': theme.borderWidths.borderWidth20,
+      '--docsearch-highlight-color': theme.backgroundColors.colorBackgroundPrimaryStronger,
+      '--docsearch-muted-color': theme.textColors.colorTextWeak,
+      '--docsearch-container-background': theme.backgroundColors.colorBackgroundOverlay,
+      '--docsearch-logo-color': theme.textColors.colorTextWeak,
+      '--docsearch-modal-width': theme.sizes.size70,
+      '--docsearch-modal-height': '600px',
+      '--docsearch-modal-background': theme.backgroundColors.colorBackgroundBody,
+      '--docsearch-modal-shadow': theme.shadows.shadowCard,
+      '--docsearch-searchbox-height': '46px',
+      '--docsearch-searchbox-background': theme.backgroundColors.colorBackgroundInverse,
+      '--docsearch-searchbox-focus-background': theme.backgroundColors.colorBackgroundBody,
+      '--docsearch-searchbox-shadow': theme.shadows.shadowFocus,
+      '--docsearch-hit-height': '56px',
+      '--docsearch-hit-color': theme.textColors.colorText,
+      '--docsearch-hit-active-color': '#fff',
+      '--docsearch-hit-background': theme.backgroundColors.colorBackgroundBody,
+      '--docsearch-hit-shadow': 'none',
+      '--docsearch-key-gradient': theme.backgroundColors.colorBackgroundStrong,
```

Comment:
Can you make this colorBackground instead? Contrast on the key shortcut is pretty `bad` right now in dark theme.